db/deploy/mutations/create_rfi.sql

```diff
@@ -0,0 +1,30 @@
+-- Deploy ccbc:mutations/create_rfi to pg
+
+begin;
+
+create or replace function ccbc_public.create_rfi(row_id int)
```

Comment:
Nit: For clarity, can we make this:  
```suggestion
create or replace function ccbc_public.create_rfi(application_row_id int)
```

Reply:
As Marcel stated, this is missing adding in the json data. Please add that in there

Reply:
Done